docs/components/label.md

```diff
@@ -77,18 +77,41 @@ Use the `for` attribute to bind a label to an input element for accessibility.
 <KInput id="service"/>
 ```
 
+## Slots
+
+- `tooltip` - Rather than using the `help` or `info` props, if you want to style the tooltip content you can use this slot.
```

Comment:
```suggestion
- `tooltip` - Rather than using the `help` or `info` props, if you need to utilize HTML in the tooltip, you may use the `tooltip` slot.
```